internal/handlers/sqlite/msg_aggregate.go

```diff
@@ -394,23 +396,42 @@ func processStagesStats(ctx context.Context, closer *iterator.MultiCloser, p *st
 		if err != nil {
 			return nil, lazyerrors.Error(err)
 		}
+
+		var list *backends.ListCollectionsResult
+
+		if list, err = p.db.ListCollections(ctx, new(backends.ListCollectionsParams)); err != nil {
```

Comment:
Same as for `msg_collstats`: maybe we don't need to query all the collections if we need only one.

Reply:
`ListCollections` does not query anything